ocs_ci/deployment/hosted_cluster.py

```diff
@@ -493,31 +534,6 @@ def deploy_dependencies(
             self.update_hcp_binary()
         if deploy_mce and not deploy_acm_hub:
             self.deploy_mce()
-            self.validate_mce_deployment()
-
-        provider_ocp_version = str(
-            get_semantic_version(get_ocp_version(), only_major_minor=True)
-        )
-        latest_released_ocp_version = str(
-            get_semantic_version(get_latest_release_version(), only_major_minor=True)
-        )
-
-        if provider_ocp_version > latest_released_ocp_version:
```

Comment:
This looks ok to me. If the workaround is needed for older version then we can use release-4.17 branch.

Reply:
we don't need this workaround anymore, because it was not stable at all.
Now we have option to deploy Hyperconverged from the start, in aregular way we do it will all other dependencies, instead of installing ACM -> turning off imbedded Hyperconverged -> installing Hyperconverged upstream.
So this code is redundant and not stable, hence removed